tesseract/tesseract_planning/tesseract_motion_planners/src/trajopt/config/trajopt_planner_config.cpp

```diff
@@ -0,0 +1,15 @@
+#include "tesseract_motion_planners/trajopt/config/trajopt_planner_config.h"
+
+namespace tesseract_motion_planners
+{
+TrajOptPlannerConfig::TrajOptPlannerConfig(trajopt::TrajOptProb::Ptr prob_) { prob = prob_; }
```

Comment:
What about merging this with the base class?

Reply:
For some reason I thought having an abstract base class was a better idea, but I think it makes sense to merge them